server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
I'm around @tombuildsstuff. I've got a conference on the 5th December but I'm fine apart from that.